Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -335,8 +363,8 @@ namespace CGAL {
   CGAL_assertion(nb_facets == expected_nb_facets);
   CGAL_USE(nb_facets); CGAL_USE(expected_nb_facets);
 
-
-  internal::Mesh_3::split_in_polylines(graph, polylines, K());
+  CGAL::Mesh_3::Polyline_visitor<Point_3, Graph> visitor(polylines, graph);
+  CGAL::split_graph_into_polylines(graph, visitor);
```

Comment:
I understand know: the graph is copied, but not the properties of the vertices (because the properties of the new graph is just the `vertex_descriptor`s of the corresponding vertices in the original graph.

For Mesh_3 that should not be noticeable, compared to all the memory and CPU used by the algorithm that samples the 1D-features.